google-cloud-talent/CHANGELOG.md

```diff
@@ -1 +1,5 @@
 # Release History
```

Comment:
It seems that the generated `CHANGELOG.md` needs an extra blank line in order to play well with releasetool. Can you manually add one here for now?

Reply:
just saw this. I'll fix it